pkg/apiclient/repository/repository.pb.go

```diff
@@ -320,7 +320,9 @@ type RepoAccessQuery struct {
 	// The type of the repo
 	Type string `protobuf:"bytes,9,opt,name=type,proto3" json:"type,omitempty"`
 	// The name of the repo
-	Name                 string   `protobuf:"bytes,10,opt,name=name,proto3" json:"name,omitempty"`
+	Name string `protobuf:"bytes,10,opt,name=name,proto3" json:"name,omitempty"`
+	// Whether helm-oci support should be enabled for this repo
+	EnableOci            bool     `protobuf:"varint,11,opt,name=enableOci,proto3" json:"enableOci,omitempty"`
```

Comment:
golang coding standard sugguests that acronyms to have a consistent case.  For example, `EnableOci` should be `EnabledOCI`.  See 
https://github.com/golang/go/wiki/CodeReviewComments#initialisms